geometric_kernels/kernels/geometric_kernels.py

```diff
@@ -266,3 +290,6 @@ def K(
     def K_diag(self, params, state, X: B.Numeric, **kwargs) -> B.Numeric:
         """Compute the kernel via integration of heat kernel"""
         return self.kernel(params, X, diag=True)
+
+    def feature_map(self, params, state, **kwargs):
```

Comment:
Here, we approximate Matern as a sum of heat kernels with various length scales. The straightforward way to get a feature map is to just concatenate feature maps of these heat kernels with various length scales. Although the result could be huge and thus this is not very efficient. But so are integrated kernels :)